### ISSUE TITLE
3.19.2 patch release

### ISSUE THREAD

**Issue by uwedeportivo** [sourcegraph/sourcegraph#13495]
# 3.19.2 Patch Release

**Attention developers:** Add pending changes to this checklist, cherry-pick the relevant commits onto branch `3.19`. **Only check off items if the relevant PR/commits have been cherry-picked into the branch**:

- [x] https://github.com/sourcegraph/sourcegraph/commit/a97f81b0f79535253bd7eae6c30d5c91d48da5ca
- [x] https://github.com/sourcegraph/sourcegraph/commit/1c22d8ce13c149b3fa3a7a26f8cb96adc89fc556
- [x] https://github.com/sourcegraph/sourcegraph/commit/d8d61b43c0f0d229d46236f2f128ca0f93455172

## Release sourcegraph/server

- [x] Push the branch [`3.19`](https://github.com/sourcegraph/sourcegraph/tree/3.19) with your cherry-picked commit(s) and make sure CI passes.
- [x] Push a release candidate tag:
    ```
    git checkout '3.19'
    git tag -a 'v3.19.2-rc.1' -m 'v3.19.2-rc.1' && git push origin 'v3.19.2-rc.1'
    ```
- [x] If CI passes, push the release tag:
    ```
    git checkout '3.19'
    git tag -a 'v3.19.2' -m 'v3.19.2' && git push origin 'v3.19.2'
    ```
- [x] Wait for the final Docker images to be available at https://hub.docker.com/r/sourcegraph/server/tags.
- [x] Run the old and new images at least three times to make sure it starts:
    ```
    # 1. Answer YES to delete /tmp/sourcegraph with the old image
    IMAGE=sourcegraph/server:3.19 ./dev/run-server-image.sh
    
    # 2. Answer NO to delete /tmp/sourcegraph with the new image
    IMAGE=sourcegraph/server:3.19.2 ./dev/run-server-image.sh
    
    # 3. Answer YES to delete /tmp/sourcegraph with the new image
    IMAGE=sourcegraph/server:3.19.2 ./dev/run-server-image.sh
    ```

## Release Kubernetes deployment

In [deploy-sourcegraph](https://github.com/sourcegraph/deploy-sourcegraph):

- [x] Wait for Renovate to open a PR named **"Update Sourcegraph Docker images"** and merge that PR ([example](https://github.com/sourcegraph/deploy-sourcegraph/pull/199) and note Renovate may have merged it automatically).
- [x] Wait for Renovate to open a PR named **"Update Sourcegraph Prometheus / Grafana Docker images"** and merge that PR (note Renovate may have merged it automatically).
- [x] Cherry-pick the image tag update commits from `master` onto `3.19` branch. Then push the release tag:
    ```
    git checkout 3.19
    git pull
    git cherry-pick <commit0> <commit1> ... # all relevant commits from the master branch
    git push 3.19
    git tag v3.19.2
    git push origin v3.19.2
    ```

## Release Docker Compose

- [x] Release Docker Compose by following [these instructions](https://github.com/sourcegraph/deploy-sourcegraph-docker/blob/master/RELEASING.md)

## Update the docs

- [x] Open and merge PRs that publish the release (the PRs created by this command must be merged manually):
  ```
  yarn run release release:publish 3.19.2
  ```
- [x] Add an entry to https://docs.sourcegraph.com/admin/updates/kubernetes indicating the steps required to upgrade.
- [x] Cherry pick the release-publishing PR from sourcegraph/sourcegraph@master into the release branch.  
- [x] Create a new section for the patch version in the changelog. Verify that all changes that have been cherry picked onto the release branch have been moved to this section of the [CHANGELOG](https://github.com/sourcegraph/sourcegraph/blob/master/CHANGELOG.md) on `master`.
- [x] Post a reply in the #dev-announce thread to say that the release is complete.

**Comment by keegancsmith** [comment]
I also included a97f81b0f79535253bd7eae6c30d5c91d48da5ca as a cherry-pick to simplify merge conflicts. This change does affect behaviour of saved searches. See the commit / PR for more information. Given we are including toggles to unblock any potential breakages to saved searches this should be fine to include.

I have edited the description to include this commit and have pushed the cherry-picks onto the 3.19 branch.
